clang-tools-extra/clang-tidy/ClangTidyDiagnosticConsumer.h

```diff
@@ -81,6 +81,9 @@ class ClangTidyContext {
 
   ~ClangTidyContext();
 
+  ClangTidyContext(const ClangTidyContext &) = default;
+  ClangTidyContext &operator=(const ClangTidyContext &) = default;
```

Comment:
I think these would make more sense as `= delete;` so that we prevent accidental copies of the context object. (We want this passed around by reference or pointer only.)